### PR TITLE
add acceptable conduct for deadlines

### PR DESCRIPTION
**Pull request by connellyw**
added acceptable conduct for deadlines

### CODE REVIEW

**Review by jjumadinova — DISMISSED** [review]

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: These two bullet points could be combined to make one meaningful and concise statement.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Empty bullet point should not be kept if there isn't a statement there

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Hello @connellyw! Nice work so far. I requested few changes in the `conduct.md` file that could clarify your points a bit more. It would also be great if you could elaborate more on your PR description before it gets merged. Adding labels and assignees to the PR would also let everyone know who worked on this and what kind of changes it's making. Overall, I think this PR is a work in progress and will need some changes.

**Review by noorbuchi — CHANGES_REQUESTED** [review]
There are also some merge conflicts that should be resolved before this PR could be merged.

File: conduct.md
Comment (noorbuchi) [inline conduct.md:256]: The TODO here should not be removed. This is to notify students from other groups to also add their names

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Formatting the names like this will put them all on the same line once Markdown is rendered, It would be great to add commas between names to make this section more readable.

**Review by noorbuchi — CHANGES_REQUESTED** [review]

**Review by connellyw — COMMENTED** [review]
fixed merge errors

**Comment by connellyw** [comment]
fixed merge conflicts, and added content to deadlines

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: It would be great if you can split this bullet point into two smaller ones. Restructuring the statement can also give it more clarity

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Can you please clarify the meaning of this bullet point

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: These two points have a similar meaning. You can either clarify the difference between them or combine them together

File: conduct.md
Comment (noorbuchi) [inline conduct.md:70]: Suggestion: It might be worth including acceptable conduct for group work deadlines

File: conduct.md
Comment (jjumadinova) [inline conduct.md]: Not clear what this means. Please either clarify or remove.

**Review by jjumadinova — CHANGES_REQUESTED** [review]
See the comment on the last added bullet point.

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Hello! Please address the changes I requested in order to get this PR merged. Additionally, it would be great if you can clarify the PR description. Finally, please make sure to resolve all merge conflicts. Thank you!

File: conduct.md
Comment (mceccarelli) [reply conduct.md]: We combined them to make it more concise.

File: conduct.md
Comment (mceccarelli) [reply conduct.md]: We discussed and removed it.

Comment (mceccarelli) [reply conduct.md]: We discussed and removed it

File: conduct.md
Comment (mceccarelli) [reply conduct.md]: We edited the way it's phrased, so it reads more clearly now.

**Review by corlettim — DISMISSED** [review]

File: conduct.md
Comment (mceccarelli) [reply conduct.md:70]: We added a bullet point about group work, how and when to turn it in.

**Comment by corlettim** [comment]
Please resolve the conflicts with main

File: conduct.md
Comment (noorbuchi) [inline conduct.md:68]: Can you please explain why this point was deleted

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Nice additions! Please make sure to resolve merge conflicts and fix/respond to my comment below. Thanks!

File: conduct.md
Comment (connellyw) [reply conduct.md:68]: Was redundant with one of our points

**Review by connellyw — COMMENTED** [review]
added comment

File: conduct.md
Comment (noorbuchi) [reply conduct.md:68]: I personally don't think it was redundant but I'm fine with your decision to remove it. Once merge conflicts are resolved, I will approve this PR

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: An empty line should be kept at the end of the file

File: conduct.md
Comment (noorbuchi) [inline conduct.md:76]: The deleted changes are from other groups. I think that they should be kept since they were agreed on by others.

**Review by noorbuchi — CHANGES_REQUESTED** [review]

**Review by enpuyou — DISMISSED** [review]

**Review by corlettim — DISMISSED** [review]

**Comment by enpuyou** [comment]
Don't forget to resolve the merge conflicts before the PR could be merged!

File: conduct.md
Comment (noorbuchi) [reply conduct.md:256]: Since this will likely be the last PR and now that all students had a chance to add their names, you could remove this TODO. Sorry for the confusion

**Review by noorbuchi — CHANGES_REQUESTED** [review]

**Review by enpuyou — DISMISSED** [review]
The PR looks good. Ready to merge once the TODO issue is addressed. Thanks!

**Review by noorbuchi — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]

**Review by corlettim — APPROVED** [review]

**Review by munzekm — APPROVED** [review]

**Pull request merged**
